source/docs/essentials/adapt-theme-to-your-brand.md

```diff
@@ -0,0 +1,152 @@
+---
+id: adapt-theme-to-your-brand
+title: Adapt the theme to your brand
+---
+
+When you start a new project, one of the first things that you will need to do
+is to adapt the look and feel of Front-Commerce to match your brand's style.
+
+Obviously, it is not something that can be done in just a few minutes because
+your identity shines in the details. However, you can do all the heavy lifting
+pretty quickly by customizing what is called **Design Tokens**.
+
+And it's a great way to familiarize yourself with Front-Commerce!
+
+## Get your Design Tokens
+
+This concept is often closely related to Design Systems. Basically Design Tokens
+are the _core styles_ of your design: colors, font families, font sizes,
+borders, shadows, etc. Together they are what make your brand unique.
+
+There even exist tools that extract those tokens from existing websites. In this
+example, we will use [CSS Stats](https://cssstats.com/) to extract our Design
+Tokens.
+
+<figure>
+<img alt="CSS Stats lists all the 62 colors and 59 unique background colors of Smashing Magazine" src="/docs/assets/smashingmagazine-cssstats.png" />
+<figcaption>Here is an example with the awesome Smashing Magazine website</figcaption>
+</figure>
+
+<blockquote class="resource">
+If you want to learn more about it, you can have a look at
+[Design tokens for dummies](https://uxdesign.cc/design-tokens-for-dummies-8acebf010d71)
+which is a very nice introduction.
+</blockquote>
+
+## Apply these tokens to your theme
+
+Now that we've got our Design Tokens, let's apply them to Front-Commerce's
+theme.
+
+Since we use the Atomic Design principles, the tokens are within the atoms in
+`/web/theme/components/atoms`. In this guide, we'll focus on the colors and the
+typography settings. But feel free to go further and edit buttons, form inputs,
+etc.
+
+In order to style our HTML, we use Sass, the well-known CSS preprocessor. Thus,
+the design tokens often translate to Sass variables.
+
+
+### Colors
+
+For instance, if we want to edit the colors of our application, we need to override the one defined in the core.
+To do so:
+1. override the `components/atoms/Colors/_colors.scss` theme file in your theme:
```

Comment:
The root is different than the one used in explaining atoms.

I'd suggest to first explain where the files related  to the `theme` are. Then, I'd suggest to always mention the `theme` folder in the tokens customization (color, typography).

Shouldn't we move around the base theme of front-commerce to match the actual structure of a module before merging 1.0.0-alpha.1 ? It would make the docs clearer imo.